src/Components/SavedSearchAlert/ConfirmationArtworksGrid.tsx

```diff
@@ -0,0 +1,104 @@
+import { Flex, SkeletonText, Spacer, Text } from "@artsy/palette"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { FC } from "react"
+import { graphql } from "react-relay"
+import {
+  ConfirmationArtworksGridQuery,
+  ConfirmationArtworksGridQuery$data,
+} from "__generated__/ConfirmationArtworksGridQuery.graphql"
+import ArtworkGrid, {
+  ArtworkGridPlaceholder,
+} from "Components/ArtworkGrid/ArtworkGrid"
+import { SearchCriteriaAttributes } from "Components/SavedSearchAlert/types"
+import { useTranslation } from "react-i18next"
+
+interface ConfirmationArtworksProps {
+  artworksConnection: ConfirmationArtworksGridQuery$data["artworksConnection"]
+}
+
+const ConfirmationArtworks: FC<ConfirmationArtworksProps> = ({
+  artworksConnection,
+}) => {
+  const { t } = useTranslation()
+  const artworksCount = artworksConnection?.counts?.total
+
+  return (
+    <Flex flexDirection="column">
+      <Text variant="sm-display" color="black60">
+        {t("createAlertModal.confirmationStep.artworksMatchCriteria", {
+          artworksCount,
+        })}
+      </Text>
+      <Text variant="sm-display" color="black60">
+        {t("createAlertModal.confirmationStep.seeOurTopPicks")}
+      </Text>
+
+      <Spacer y={2} />
+
+      <ArtworkGrid artworks={artworksConnection!} columnCount={2} />
+    </Flex>
+  )
+}
+
+export const ConfirmationArtworksGridQueryRenderer: FC<SearchCriteriaAttributes> = props => {
+  return (
+    <SystemQueryRenderer<ConfirmationArtworksGridQuery>
+      placeholder={<ContentPlaceholder />}
+      query={graphql`
+        query ConfirmationArtworksGridQuery($input: FilterArtworksInput) {
+          artworksConnection(input: $input) {
+            counts {
+              total
+            }
+            ...ArtworkGrid_artworks
+          }
+        }
+      `}
+      variables={{
+        input: {
+          first: 10,
```

Comment:
in ticket:

> 10 works for all devices except tablet, in which case we’d display 20.

Couldn't find anything similar to it in the codebase. What would be a correct way to determine if user uses tablet?

Reply:
Not a thing. Not sure what would make tablet's different than desktop :)

Reply:
I mean you *could* detect a combination of screen dimensions and touch. But we shouldn't, I think the design is just specified oddly.

Reply:
I've just remembered that this is not a requirement from designer, rather something we came up with during eng refinement. We though that 10 artworks won't fill modal fully on tables in vertical mode. I've just checked on my ipad and all looks fine, so let's use 10 for all devices